### DISCUSSION TITLE
Fatura NO return

### DISCUSSION THREAD

**Discussion by Looden** [mlevent/fatura#22] [General]
$ettn = $invoice->getUuid();
seklinde oldgu gibi, fatura numarasini da kesildikten sonra donuyor mu yoksa $gib->getDocument motedunu mu kullanmaliyiz.

**Comment by mlevent** [comment]
getDocument veya getLastDocument metodlarından birini kullanmalısınız. Uuid bilgisi elinizde olduğundan getDocument metodunu kullanmanız daha mantıklı olacaktır.
